app/V1Module/presenters/EmailsPresenter.php

```diff
@@ -153,19 +167,18 @@ public function actionSendToGroupMembers(string $groupId) {
       $users = array_merge($users, $group->getAdmins());
     }
 
-    $emails = array_map(function (User $user) {
-      return $user->getEmail();
-    }, $users);
-
     // user requested copy of the email to his/hers email address
-    if ($toMe && !in_array($user->getEmail(), $emails)) {
-      $emails[] = $user->getEmail();
+    if ($toMe && !in_array($user, $users)) {
```

Comment:
I do not like using in_array on objects (actually, I do not like the function at all, since it requires O(N) time). Comparison of two object invokes == operator on each property, which may not even work since these objects are Doctrine entities if I am not mistaken (there is no guarantee, that they may not have some internal members not related to the user's data). I believe that the only correct way here is to compare user IDs only.